source/regions/common-scenarios.rst

```diff
@@ -70,25 +70,25 @@ How do I unprotect all enchantment tables?
 
 In the :doc:`../config`, adjust the ``interaction-whitelist`` setting and add the enchantment table. Get the proper names that you can use from `Bukkit's Material <https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/Material.html>`_.
 
-How do I allow mining but prevent block breaking?
+How do I allow mining but prevent block placing?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Set the ``block-break`` flag to ``allow``::
 
-* /rg flag mining_area block-break allow
+    /rg flag mining_area block-break allow
 
 How do I allow players to read lectern books, but not take them?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 First, make sure you are on WorldGuard 7.0.1+. Earlier versions of WorldGuard released before 1.14 (and thus before lecterns) existed.
 
-* Set the ``interact`` flag to ``allow``: ``/rg flag <region> interact allow``
-* Set the ``chest-access`` flag to ``deny``, at least for non-members: ``/rg flag <region> chest-access -g nonmembers deny``
+* Set the ``use`` flag to ``allow``: ``/rg flag <region> use allow``
+* Set the ``chest-access`` flag to ``deny``, at least for non-members (note that this is the default): ``/rg flag <region> chest-access -g nonmembers deny``
```

Comment:
```suggestion

```
This was written before I separated the use and chest-access flags, it's no longer necessary at all.